using-timescaledb/continuous-aggregates.md

```diff
@@ -323,24 +323,19 @@ aggregate, such as views that are built on top of the continuous aggregate view.
 
 
 ### Dropping Data with Continuous Aggregates Enabled [](dropping-data)
-When dropping data in a raw hypertable using the [`drop_chunks` function][api-drop-chunks]
-that has a continuous aggregate created on it, we must specify the `cascade_to_materializations`
-argument to the `drop_chunks` call. A value of `true` will cause the continuous aggregate 
-to drop all data associated with any chunks dropped from the raw hypertable. A value
-of `false` will retain data in the continuous aggregate while dropping only the
-raw data.
-
->:TIP: When dropping data from the raw hypertable while retaining data
-on a continuous aggregate, the `older_than` parameter to `drop_chunks`
-has to be longer than the `timescaledb.ignore_invalidation_older_than`
-parameter on the continuous aggregate. That is because we cannot
-process invalidations on data regions where the raw data has been
-dropped.
-
-
-The same argument must also be supplied to the [`add_retention_policy`
-function][api-add-retention] when creating a data retention policy for a
-hypertable with a continuous aggregate.  
+Note that if any still-refreshing (more recent than start_offset) part of the
+continuous aggregate is dropped via a [retention policy][api-add-retention] or
+direct [`drop_chunks`][api-drop-chunks] call, the aggregate will be updated to
+reflect the loss of data. For this reason, if it desired to retain the continuous
```

Comment:
I think you need to have something along the lines "If there are outstanding invalidations in the hypertable, ....".

Reply:
... also, what happens if the user does an explicit `DELETE`? I assume nothing specific happens, but it is useful to mention to indirectly suggest that `drop_chunks` function should be used.